tck/pom.xml

```diff
@@ -98,6 +93,11 @@
             <artifactId>jackson-jaxrs-json-provider</artifactId>
             <version>2.8.2</version>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-api</artifactId>
+            <version>0.30.0</version>
```

Comment:
Is this version accurate for opentracing?

Reply:
This is correct. 0.31.0 is still an RC1, so not using that yet.